### PR TITLE
fix: update latest docker image

### LINKED ISSUE #1140: Un-support statement type: SystemVariableAssignment
Issue tracker is **ONLY** used for reporting bugs. New features should be discussed on our discussion
使用docker 镜像4pdosc/openmldb:latest ，启动容器，运行案例1：出租车行车时间预测，出现Un-support statement type: SystemVariableAssignment的错误

image

### PR DESCRIPTION
**Pull request by dl239**
update latest docker image

### CODE REVIEW

File: .github/workflows/openmldb-docker.yml [hunk 1]
```diff
@@ -68,4 +70,5 @@ jobs:
           platforms: linux/amd64
           tags: |
             ${{ env.DOCKERHUB_TAG }}
-          build-args: OPENMLDB_VERSION=${{ github.event.inputs.OPENMLDB_VERSION }}
\ No newline at end of file
+            ${{ env.DOCKERHUB_LATEST }}
```
Comment (aceforeverd) [inline .github/workflows/openmldb-docker.yml:73]: strange, why not just pass `OPENMLDB_VERSION` = `latest`

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by zhanghaohit — APPROVED** [review]

File: .github/workflows/openmldb-docker.yml (hunk 1 above)
Comment (dl239) [reply .github/workflows/openmldb-docker.yml:73]: if pass `OPENMLDB_VERSION`, we should trigger docker release twice. And we may forget to release the latest version

Comment (aceforeverd) [reply .github/workflows/openmldb-docker.yml:73]: what is the meaning of `latest` ?  If that always link to a const number tag, it may not necessary.

Comment (aceforeverd) [reply .github/workflows/openmldb-docker.yml:73]: and if we are gona patch a previous version, the latest tag also overrides

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**